source/developer_manual/development_tips/valgrind.rst

```diff
@@ -0,0 +1,77 @@
+========
+Valgrind
+========
+
+The most useful Valgrind parameters:
+
+- ``–-vgdb=no``: Needed on some systems to avoid problems.
+
+-  ``--num-callers=n``: Display last n functions calls in the backtrace for each error.
+
+-  ``--leak-check=full``: Show also unfreed memory as being leaked.
+
+-  ``--trace-children=yes``: Sometimes you might want to trace also the forked child processes.
+
+-  ``--suppressions=<path>``: Suppressions for things that can't be fixed. Usually in external libraries.
+
+-  ``-q``: Quiet - don't log about initialization etc.
```

Comment:
we should include `--keep-debuginfo=yes` here as well with version number information.

Reply:
it's mentioned later, but it should be mentioned here too.